frontend/src/pages/LogsPage/useGetLogs.ts

```diff
@@ -85,6 +87,7 @@ export const useGetLogs = ({
 		GetLogsQuery,
 		GetLogsQueryVariables
 	>({
+		skip: disablePolling,
```

Comment:
one thing i noticed from click testing is that the polling query still uses a fixed time range (that looks 1 hour into the future, so if the page is open for more than 1 hour, it won't query new data). might be something we should change, though don't necessarily need to do this in this pr

Reply:
Yes, I saw this. I think ideally it would use the current time as the end date, which should hopefully be a quick fix.

This would be an issue if the user did not receive any logs/traces for an hour, but we would show the "X more logs" that were fetched in the time frame. Then when the user clicks it, we would fetch the logs with the "Last Y time" using the current time as the end date.